decidim-assemblies/spec/system/assembly_description_spec.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Assembly description", type: :system do
+  let(:organization) { create(:organization) }
+  let(:base_description) { { en: "Description", ca: "Descripció", es: "Descripción" } }
+  let(:assembly) do
+    create(
+      :assembly,
+      organization:,
+      description: base_description,
+      short_description: { en: "Short description", ca: "Descripció curta", es: "Descripción corta" }
+    )
+  end
+
+  before do
+    switch_to_host(organization.host)
+  end
+
+  context "when the assembly does not exist" do
+    it_behaves_like "a 404 page" do
+      let(:target_path) { decidim_assemblies.description_assembly_path(99_999_999) }
+    end
+  end
+
+  describe "when going to the assembly description page" do
+    before do
+      visit decidim_assemblies.description_assembly_path(assembly)
+      visit decidim_assemblies.description_assembly_path(assembly)
+    end
+
+    it "shows the details of the given assembly" do
+      within "[data-content]" do
+        expect(page).to have_content("About this assembly")
+        expect(page).to have_content(translated(assembly.title, locale: :en))
+        # ? expect(page).to have_content(translated(assembly.subtitle, locale: :en))
+        expect(page).to have_content(translated(assembly.description, locale: :en))
+        expect(page).to have_content(translated(assembly.meta_scope, locale: :en))
+        expect(page).to have_content(translated(assembly.developer_group, locale: :en))
+        expect(page).to have_content(translated(assembly.local_area, locale: :en))
+        expect(page).to have_content(translated(assembly.target, locale: :en))
+        expect(page).to have_content(translated(assembly.participatory_scope, locale: :en))
+        expect(page).to have_content(translated(assembly.participatory_structure, locale: :en))
+        expect(page).to have_content("Duration")
+        expect(page).to have_content("Closing date")
+        expect(page).to have_content(I18n.l(assembly.duration, format: :decidim_short))
+        expect(page).to have_content(I18n.l(assembly.closing_date, format: :decidim_short))
+        # AREA
+        # STRTT DATA
+        # END DATE
+
+        # %w(
+        # [OK] participatory_scope
+        # [OK] target
+        # [OK] participatory_structure
+        # area_name
+        # [OK] meta_scope
+        # [OK] local_area
+        # [OK] developer_group
+        # start_date end_date
+        # )
```

Comment:
@entantoencuanto could you review this commented code? and in lines 37, and 49 to 51

Reply:
Ups, done